mzLib/FlashLFQ/ResultsReading/IdentificationAdapter.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Readers.ExternalResults.BaseClasses;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FlashLFQ.ResultsReading
+{
+    public static class IdentificationAdapter
+    {
+        public static List<Identification> MakeIdentifications(IQuantifiableResultFile quantifiable)
```

Comment:
This would not be a textbook 'adapter' in its current state. I would recommend creating an MzLibExtensions class, then making this an extension method 
public static List<Identification> MakeIdentifications(this IQuantifiableResultFile quantifiable)

Feel free to ignore this recommendation by resolving the comment